tests/unit/amo/components/TestAddonAdminLinks.js

```diff
@@ -22,6 +22,7 @@ import {
 
 describe(__filename, () => {
   let store;
+  const id = 123;
   const slug = 'some-slug';
```

Comment:
Is the `slug` still used in this test file?

Reply:
Yes, for the link to devhub.